Backend/app/rider-platform/rider-app/Main/src/Beckn/ACL/Search.hs

```diff
@@ -35,7 +34,7 @@ buildSearchReqV2 ::
   m Spec.SearchReq
 buildSearchReqV2 res@DSearch.SearchRes {..} = do
   bapUri <- Utils.mkBapUri merchant.id
-  bapConfig <- QBC.findByMerchantIdDomainAndVehicle merchant.id "MOBILITY" AUTO_RICKSHAW >>= fromMaybeM (InternalError $ "Beckn Config not found for merchantId:-" <> show merchant.id.getId <> ",domain:-MOBILITY,vehicleVariant:-" <> show AUTO_RICKSHAW) -- get Vehicle Variatnt here
+  bapConfig <- QBC.findByMerchantIdDomainandMerchantOperatingCityId merchant.id "MOBILITY" res.merchantOperatingCityId >>= fromMaybeM (InternalError $ "Beckn Config not found for merchantId:-" <> show merchant.id.getId <> "merchantOperatingCityId:-" <> show merchantOperatingCityId.getId)
```

Comment:
For backward compatibility, do findAll and list to maybe.
Add comment to fix this after release